hazelcast/src/main/java/com/hazelcast/map/impl/operation/BasePutOperation.java

```diff
@@ -47,6 +47,7 @@ protected void afterRunInternal() {
         invalidateNearCache(dataKey);
         publishWanUpdate(dataKey, value);
         evict(dataKey);
+        super.afterRunInternal();
```

Comment:
Currently `super.afterRunInternal()` has empty body so it does not matter, but is there specific reason for deferring this call to the end of the method? 🤔

Reply:
I just followed the conventions to call it in the end. I noticed that for some methods the call is missing.